daemon/daemon.go

```diff
@@ -312,6 +315,25 @@ func (d *Daemon) fmtPolicyEnforcement() string {
 	return fmt.Sprintf("#undef %s\n", endpoint.OptionSpecPolicy.Define)
 }
 
+// Must be called with d.conf.EnablePolicyMU locked.
+func (d *Daemon) writePreFilterHeader(dir string) error {
```

Comment:
This comment is not true since ad05d246c3217ef8af560ce49deaafbe85a25e97 fixed up the policy enforcement mutex locking logic, so you can remove it.